rmw_fastrtps_cpp/src/rmw_client.cpp

```diff
@@ -215,8 +225,9 @@ rmw_create_client(const rmw_node_t * node,
         _unregister_type(participant, info->response_type_support_, info->typesupport_identifier_);
       }
     } else {
-      fprintf(stderr,
-        "[rmw_fastrtps] leaking type support objects because node impl is null\n");
+      RCUTILS_LOG_ERROR_NAMED(
+        "rmw_fastrps_cpp",
```

Comment:
arg I had to chose an already typoed line for my copy-n-paste :confounded: thanks for catching it